src/pages/Admin/Venue/VenueWizard/Create/VenueWizardCreate.types.ts

```diff
@@ -0,0 +1,7 @@
+import { WizardState } from "../redux";
+
+export interface VenueWizardCreateProps {
+  state: WizardState;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  dispatch: any;
```

Comment:
Remove `any` Dispatch type should exist in react or react-redux

Reply:
Depending on what the 'meaning' of this `dispatch` is, we may just not need it. If it is just the normal redux `dispatch`, then we should probably just use the `useDispatch` hook in the component where we need it, rather that pass it around as props.